DEVELOP.rst

```diff
@@ -1,84 +1,94 @@
-========================
-Crate-Python Development
-========================
+===============
+Developer Guide
+===============
 
+Setup
+=====
 
-Development Setup
-=================
+This project uses buildout_ to set up the development environment.
 
-To get a development environment crate-python uses `buildout
-<https://pypi.python.org/pypi/zc.buildout/2.2.1>`_
+To start things off, run::
 
-Run `bootstrap.py`::
+    $ python bootstrap.py
 
-    python bootstrap.py
+Then, run::
 
-And afterwards run buildout::
-
-    ./bin/buildout -N
+    $ ./bin/buildout -N
 
 Running Tests
 =============
 
-The tests are run using the `zope.testrunner
-<https://pypi.python.org/pypi/zope.testrunner/4.4.1>`_::
+The tests are run using the zope.testrunner_::
 
-    ./bin/test
+    $ ./bin/test
 
 This will run all tests using the python interpreter that was used to
 bootstrap buildout.
 
-In addition to that it is also possible to run the test case against multiple
-python interpreter using `tox <http://testrun.org/tox/latest/>`_::
+You can run the tests against multiple Python interpreters with tox_::
 
-    ./bin/tox
+    $ ./bin/tox
 
-This required the interpreters `python2.7`, `python3.3` and `pypy` to be
-available in `$PATH`. To run against a single interpreter tox can also be
-invoked like this::
+To do this, you will need ``python2.7``, ``python3.3``, and ``pypy`` on your
+``$PATH``.
 
-    ./bin/tox -e py33
+To run against a single interpreter, you can also do::
 
-Note: Before running tests make sure to stop all crate instances which
-transport port is listening on port 9300 to avoid side effects with the test
-layer.
+    $ ./bin/tox -e py33
 
+*Note*: before running tests make sure to stop all CrateDB instances which
+listening on port 9300 to avoid side effects with the test layer.
```

Comment:
we do use port 9300 as default for anything. We can say that a user has to assure that there are no locally running instances which listen on the default crate transport port 4300.